Eventing Docs updates for 6.6 (#1474)
couchbase/docs-server: modules/eventing/pages/eventing-handler-genericFlatten.adoc

= Function: genericFlatten
:page-edition: Enterprise Edition
:tabs:

*Goal*: Recursively and Generically Flatten a document for integration with a non-NOSQL RDBMS.

* This function *genericFlatten* shows how a document can be flattened into a RDBMS importable record.

> * Requires a metdata bucket, a source bucket, and a destination bucket aliases to "dst_bkt" in mode read+write.

Reviewer: ```suggestion
* Requires a metadata bucket, a source bucket, and a destination bucket aliased to "dst_bkt" in mode read+write.
```

> * Will operate on all documents with doc.type === "my_data_to_flatten".

Reviewer: ```suggestion
* Will operate on all documents where doc.type === "my_data_to_flatten".
```

> * Will write transformed or flatted documents to the destination bucket with the same type and id.

Reviewer: ```suggestion
* Will write transformed or flattened documents to the destination bucket with the same type and id.
```

For consistency with terminology below
